docusign/1.0/antora.yml

```diff
@@ -0,0 +1,18 @@
+name: docusign-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: DocuSign Connector
```

Comment:
The certification guys suggested to change the name to DocuSign Connector - Mule 4

Reply:
I don't know about this. Mule has never been part of the naming convention for connectors. I know it's part of the artifact id, so maybe for line 1, it could match the artifact ID, ie, mule4-docusign-connector.